build/three.js

```diff
@@ -9106,17 +9106,17 @@
 		clone: function clone() {
 			/*
 			 // Handle primitives
-				 const parameters = this.parameters;
```

Comment:
Please do not include build files in the PR.

Reply:
ok